tide_grant.install

```diff
@@ -213,3 +215,83 @@ function tide_grant_update_8011() {
     }
   }
 }
+
+/**
+ * Adds field metatags to tide_grant.
+ */
+function tide_grant_update_8012() {
+  // Adding the field from config install.
+  $metaTags = 'field.field.node.grant.field_metatags';
+  $config_path = drupal_get_path('module', 'tide_grant') . '/config/install';
+  $source = new FileStorage($config_path);
+  // Obtain the storage manager for field instances.
+  // Create a new field instance from the yaml configuration and save.
+  \Drupal::entityManager()->getStorage('field_config')
+    ->create($source->read($metaTags))
+    ->save();
+
+  // Adding the field to display form.
+  $entity_form_display = Drupal::entityTypeManager()->getStorage('entity_form_display')->load('node.grant.default');
+  if ($entity_form_display) {
+    $entity_form_display->setComponent('field_metatags', [
+      'weight' => 13,
+      'settings' => ['sidebar' => TRUE],
+      'third_party_settings' => [],
+      'type' => 'metatag_firehose',
+      'region' => 'content',
+    ]);
+  }
+  $entity_form_display->save();
+
+  // Adding the field to display view.
+  $entity_view_display = Drupal::entityTypeManager()->getStorage('entity_view_display')->load('node.grant.default');
+  if ($entity_view_display) {
+    $entity_view_display->setComponent('field_metatags', [
+      'weight' => 128,
+      'label' => 'above',
+      'settings' => [],
+      'third_party_settings' => [],
+      'type' => 'metatag_empty_formatter',
+      'region' => 'content',
+    ]);
+  }
+  $entity_view_display->save();
+}
+
+/**
+ * Update sitemap indexing and search metatag to noindex.
+ */
+function tide_grant_update_8013() {
+  // Getting only the grant content type node ids.
+  $nids = \Drupal::entityTypeManager()
+    ->getStorage('node')
+    ->getQuery()
+    ->condition('type', 'grant')
+    ->condition('uid', 0)
+    ->execute();
+  if (!empty($nids)) {
+    $nodes = Node::loadMultiple($nids);
+    foreach ($nodes as $node) {
+      // Only update the grant nodes created by anonymous user.
+      if ($node->getOwnerId() == 0) {
```

Comment:
You probably don't need to check this if you're excluding uid 0 anyway but not a bad check to have.